pkg/config/config.go

```diff
@@ -34,8 +34,8 @@ type ContainerdConfig struct {
 	Snapshotter string `toml:"snapshotter" json:"snapshotter"`
 	// DefaultRuntime is the runtime to use in containerd.
 	DefaultRuntime Runtime `toml:"default_runtime" json:"defaultRuntime"`
-	// PrivilegedRuntime is a non-secure runtime used only to run trusted workloads on it
-	PrivilegedRuntime Runtime `toml:"privileged_runtime" json:"privilegedRuntime"`
+	// SecureRuntime is a secure runtime used to run untrusted workloads on it.
+	SecureRuntime Runtime `toml:"secure_runtime" json:"secureRuntime"`
```

Comment:
As @mikebrow points, to avoid confusion about what is secure or not , lets rename it to something like `runtime_untrusted_workload` (name used by cri-o). IMO SecureRuntime is a an easy name but lets keep it generic to reduce any potential confusion.